src/app/containers/Include/index.jsx

```diff
@@ -0,0 +1,27 @@
+
+const markupToRender = {
+  include: visMarkup,
+  idt2: id2Markup,
+  indepthtoolkit: idT1Markup,
+};
+
+const Include = ({ href }) => {
+  const type = href.split('/')[1];
+  return (
+    <GridItemConstrainedMedium>
+      <div dangerouslySetInnerHTML={{ __html: markupToRender[type] }} />
```

Comment:
Did we make a decision to accept the security risk of using `dangerouslySetInnerHTML`? Or are we intending to sandbox this in an iframe?

Reply:
@andrewscfc It was agreed to build on the [shadow dom approach]() done by Rhys and Chris for VJ's just to try having similar implementations for the different include types.

Reply:
Thanks for that link, the key point in the description is this:

> Shadow DOM is not supported in IE11, upon a quick look, it appears that the include falls back to an iframe.

So we `dangerouslySetInnerHTML` to include some code that uses shadow dom when possible and where not possible, uses an iframe, that sounds a pretty satisfactory approach.

I mentioned security earlier, I'm not sure this is an issue actually, we will be hitting our own endpoint on the BBC domain so we will be 'dangerously inserting' html we can be confident comes from the BBC; without a cert breach, no one should be able to impersonate our endpoint.